lib/rules/focus-order-semantics.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "focus-order-semantics",
+  "selector": "div, h1, h2, h3, h4, h5, h6, [role=heading], p, span",
+  "matches": "inserted-into-focus-order-matches.js",
+  "tags": ["cat.name-role-value", "cat.keyboard"],
```

Comment:
This should only have one category. I suggest `cat.keyboard`. Additionally, please add `best-practice` and `experimental` as well.

Reply:
Done.